addons/sale_timesheet/controllers/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import main
```

Comment:
Actually I disagree with the guidelines on controllers naming. It is like putting model into models.py . It does not help. I would always put a real name on file. In sale_timesheet it would therefore be sale_timesheet or timesheet_plan if you want to clearly separate timesheet plan for other potentiel controllers.